src/services/error/index.ts

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable instawork/error-object */
```

Comment:
I missed this on my first review - I think we'll want this error to be disabled for the entire repo (in the .eslintrc file)